test/environments.js

```diff
@@ -169,8 +168,8 @@ class ShardedEnvironment extends EnvironmentBase {
       return new core.Mongos([{ host, port }], options);
     };
 
-    const version =
-      discoverResult && discoverResult.version ? discoverResult.version.join('.') : null;
+    //const version =
```

Comment:
Why is this commented out?

Reply:
We no longer need to declare a new variable for `version` because `version` is passed into the constructor.